README.md

```diff
@@ -42,8 +42,9 @@ By default, configuration is loaded from `tslint.json`, if it exists in the curr
 tslint accepts the following commandline options:
 
     -f, --file:
-        The location of the TypeScript file that you wish to lint. This
-        option is required. Muliptle files are processed consecutively.
+        The location of the TypeScript file that you wish to lint. Multiple
+        files are processed consecutively. Files can also be tacked on to the
+        end of the command.
```

Comment:
a minor version bump is sufficient, the CLI is pretty basic and I anticipate most people using tslint via grunt/gulp wrappers anyway. let's make sure to call it out in the changelog. +1 for deprecation notice.